### PR TITLE
Editing Auditing Chapter

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10588
https://github.com/uyuni-project/uyuni-docs/pull/78

### CODE REVIEW

File: modules/administration/pages/auditing.adoc [hunk 1]
```diff
@@ -1,108 +1,87 @@
...
+Allow the task to complete before continuing with the CVE audit.
 
-An API method called `audit.listSystemsByPatchStatus` is available to run CVE audits from custom scripts.
-Details on how to use it are available in the API guide.
 
 
+.Procedure: Verifying Patch Status
+. In the {productname} {webui}, navigate to menu:Audit[CVE Audit].
+. OPTIONAL: To check the patch status for a particular CVE, type the CVE identifier in the [guimenu]``CVE Number`` field.
+. Select the patch statuses you want to look for, or leave all statuses checked to look for all.
```
Comment (keichwa) [inline modules/administration/pages/auditing.adoc]: ATM, I'm not sure about (US) English plural usuage of "status".  Traditionally, it is also "status" (with a long u, though).  Maybe, we can use "patch state" and "patch states".

File: modules/administration/pages/auditing.adoc [hunk 2]
```diff
@@ -1,108 +1,87 @@
...
-. Click the btn:[Single Run Schedule] button.
-. After some minutes, refresh the page and check that the scheduled run status is ``FINISHED``.
+For each system, the [guimenu]``Next Action`` column provides information about what you need to do to address vulnerabilities.
+If applicable, a list of candidate channels or patches is also given.
+You can also assign systems to a [guimenu]``System Set`` for further batch processing.
+
 
-A direct link is also available on the menu:Main Main[Audit > CVE Audit] page ([guimenu]``extra CVE data update``).
+You can use the {productname} API to verify the patch status of your clients.
+Use the `audit.listSystemsByPatchStatus` API method.
```
Comment (keichwa) [inline modules/administration/pages/auditing.adoc]:
```suggestion
Use the ``audit.listSystemsByPatchStatus`` API method.
```

File: modules/administration/pages/auditing.adoc [hunk 3]
```diff
@@ -1,108 +1,87 @@
...
-For each system, the [guimenu]``Next Action`` column contains suggestions on the steps to take to address the vulnerabilities.
-Under these circumstances it is either sensible to install a certain patch or assign a new channel.
-If applicable, a list of "`candidate`" channels or patches is displayed for your convenience.
+Each CVE contains an identification number, a description of the vulnerability, and links to further information.
+CVE identification numbers use the form ``CVE-YEAR-XXXX``.
 
-You can also assign systems to a [guimenu]``System Set`` for further batch processing.
+In the {productname} {webui}, navigate to menu:Audit[CVE Audit] to see a list of all clients and their current patch status.
+
+By default, the CVE data is updated at 2300 every day.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/auditing.adoc]: So I checked Chicago, and it was adamant not to use any punctuation in military time. I wondered about putting 11pm in parentheses too, but then wondered if it was really necessary? Not sure.

I'll create an issue to update the style guide :+1:

[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/auditing.adoc]: https://github.com/SUSE/spacewalk/issues/10682

File: modules/administration/pages/auditing.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/administration/pages/auditing.adoc]: My AHD (en-US) and my Macquarie (en-AU) don't give a plural. My Writers & Editors dictionary (en-GB) says the plural is `statuses`.

Comment (Loquacity) [reply modules/administration/pages/auditing.adoc]: Googling brought me here, which seems to agree that `statuses` is common en-US usage over `status`: https://en.wikipedia.org/wiki/Talk%3AStatus

**Pull request merged**